package.json

```diff
@@ -78,7 +78,7 @@
     "@babel/preset-env": "7.24.3",
     "@babel/runtime": "7.24.1",
     "@bazel/bazelisk": "1.19.0",
-    "@bazel/buildifier": "7.1.0",
+    "@bazel/buildifier": "6.4.0",
```

Comment:
Is this downgrade intentional? Was there some conflict with 7.1.0 and other updates in this PR?

Reply:
It is intentional see: https://github.com/angular/angular-cli/pull/27390/commits/86bb45ddd743693efb68fc95e30089ba8ce2784d

This dep was only updated in main